inyoka_theme_default/templates/ikhaya/events.html

```diff
@@ -0,0 +1,69 @@
+{#
+    ikhaya/events.html
+    ~~~~~~~~~~~~~~~~~~
+
+    This is the overview of the events.
+
+    :copyright: (c) 2007-2016 by the Inyoka Team, see AUTHORS for more details.
+    :license: BSD, see LICENSE for more details.
+#}
+{% extends 'ikhaya/base.html' %}
+
+{% block breadcrumb %}
+  {{ macros.breadcrumb_item(_('Events')) }}
+{% endblock %}
+
+{% block sidebar %}
+  {% call macros.sidebar(_('Calender')) %}
+    {{ macros.sidebar_item(_('Add event'), href('ikhaya', 'event', 'new')) }}
+
+    {% if not show_all %}
+      {{ macros.sidebar_item(_('Display past events'), href('ikhaya', 'events', 'all')) }}
+    {% endif %}
+
+    {% if not invisible %}
+      {{ macros.sidebar_item(_('Show only invisible events'), href('ikhaya', 'events', 'invisible')) }}
+    {% endif %}
```

Comment:
There is no link to check the invisible events. I think, we should add one to the admin section of the sidebar.